librz/arch/il/analysis_il.c

```diff
@@ -250,36 +251,96 @@ RZ_API RzAnalysisILStepResult rz_analysis_il_vm_step_while(RZ_NONNULL RzAnalysis
 		rz_analysis_il_vm_sync_from_reg(vm, reg);
 	}
 
+	RzAnalysisOp op = { 0 };
 	RzAnalysisILStepResult res = RZ_ANALYSIS_IL_STEP_RESULT_SUCCESS;
 	while (cond(vm, user)) {
 		ut64 addr = rz_bv_to_ut64(vm->vm->pc);
 		ut8 code[32] = { 0 };
 		analysis->read_at(analysis, addr, code, sizeof(code));
-		RzAnalysisOp op = { 0 };
 		rz_analysis_op_init(&op);
```

Comment:
Move it outside

Reply:
I don't get it. It seems to me that `rz_analysis_op_init` needs to be called before every call to `rz_analysis_op` and `rz_analysis_op_fini` needs to be called after every call to `rz_analysis_op`. 

So ‘Move it outside’ doesn't work? @pelijah

Reply:
Weird, `rz_analysis_op_fini()` doesn't work as expected so leave it as is.

Reply:
@XVilka We definetely should standartize `*_fini()` APIs behaviour.